src/main/scala/org/apache/spark/ml/linalg/distributed/RapidsRowMatrix.scala

```diff
@@ -91,14 +91,21 @@ class RapidsRowMatrix(
       val dense_U = DenseMatrix.zeros(n, n)
 
       val dense_S = DenseMatrix.zeros(1, n)
+      var svdDriver: Array[(DenseMatrix, DenseMatrix)] = null
       try {
-        // this is done on driver, so no task resources here, assign 0 manually.
-        RAPIDSML.calSVD(n, Cov, dense_U, dense_S, 0)
+        val rdd_tmp = rows.context.parallelize(Seq(0),1)
```

Comment:
There should be an space after ","

and better to follow the java naming conventions https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html